src/Mafia/Project.hs

```diff
@@ -0,0 +1,41 @@
+{-# LANGUAGE NoImplicitPrelude #-}
```

Comment:
Not sure if this really deserves a separate module. Is there a better place for it?